edison/static/js/policy.controller.js

```diff
@@ -127,23 +128,23 @@ function initSettingToNewPolicy() {
 }
 
 function addPolicy() {
-    policy.name = document.getElementById('policyName').value;
-    policy.room = document.getElementById('policyRoom').value;
+    policy.name = document.getElementById('policy-name').value;
+    policy.room = document.getElementById('policy-room').value;
     setCommand(policy);
     $.post('/policy', policy, function () { });
```

Comment:
I just noticed you put a POST method here that is not doing anything.
So put here a comment like I said in other conversation of ours.